### PR TITLE
Fix stop ongoing mission dialogs

### LINKED ISSUE #1529: Update pop-up message when cancelling a return to home mission
**Describe the improvement you would like to see**
When cancelling a return to home mission, the following error message appears: 
Screenshot 2024-04-08 at 10 11 33

The information is not correct and should be updated. 

**How will this change existing functionality?**
No change, just more correct information

**How will this improvement affect the current Threat Model?**
Leave blank for maintainers to fill out if you are uncertain about this.

### PR DESCRIPTION
**Pull request by mrica-equinor**
Closes #1529

### CODE REVIEW

File: frontend/src/language/no.json
Comment (prasm313) [inline frontend/src/language/no.json:232]:
```suggestion
    "Stop button pressed during return home warning text": "Du er i ferd med å stoppe et oppdrag som sender roboten tilbake til startposisjonen. Du må kjøre roboten manuelt til startposisjonen for å starte et nytt oppdrag.",
```

File: frontend/src/language/en.json
Comment (prasm313) [inline frontend/src/language/en.json:232]:
```suggestion
    "Stop button pressed during return home warning text": "You are about to stop a mission that will send the robot back to the start position. You must manually drive the robot to the start position to start a new mission.",
```

File: frontend/src/language/en.json
Comment (prasm313) [inline frontend/src/language/en.json:234]:
```suggestion
    "Stop button pressed during localization warning text": "You are about to stop a mission that localizes the robot. You will not be able to run the missions in the queue until you have successfully localized the robot."
```

File: frontend/src/language/no.json
Comment (prasm313) [inline frontend/src/language/no.json:234]:
```suggestion
    "Stop button pressed during localization warning text": "Du er i ferd med å stoppe et oppdrag som lokaliserer roboten. Du vil ikke kunne kjøre oppdragene i køen før du har lokalisert roboten."
```

**Review by prasm313 — COMMENTED** [review]
Tested: Works well!

I have some suggestions to text changes.

Would prefer if we do not call it charging station. This is also done in the "Stop button pressed warning text" at line 83.

File: frontend/src/language/en.json
Comment (prasm313) [inline frontend/src/language/en.json:1]: Change line 83 to "start position" instead of "charging station"

Comment (prasm313) [inline frontend/src/language/en.json:1]: Change line 83 to:

"You are about to stop the entire mission. If there are no more missions in the queue, the robot will return to the start position."

File: frontend/src/language/no.json
Comment (prasm313) [inline frontend/src/language/no.json:1]: Change line 83 to:

"Du er i ferd med å stoppe hele oppdraget. Hvis det ikke er flere oppdrag i kø vil roboten bli sendt tilbake til startposisjonen sin."

File: frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx
Comment (Eddasol) [inline frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx:60]:
```suggestion
    const DialogContent = (missionTaskType: TaskType) => {
```
Generally try to have PascalCase if returning a JSX element and camelCase if not

Comment (Eddasol) [inline frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx:60]: Suggest moving this function out from the function "StopMissionDialog" to avoid long functions. If moved it should also be renamed to show that it's only used for stopMissionDialog

Comment (mrica-equinor) [reply frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx:60]: I don't think it should be moved out. It is used to select the text in the stop dialogs present in this file, having it outside doesn't make sense to me

Comment (mrica-equinor) [reply frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx:60]: I think we should use this only if there's more than one argument. If it's only one then it's making hte code more crowded wihtout need

Comment (andchiind) [reply frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx]: I personally agree with @Eddasol

Comment (andchiind) [reply frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx]: But why should it be inside StopMissionDialog if it doesn't need to be? We can still have it be a function in the same file, but nesting it in the other function seems unneccessary to me

**Review by prasm313 — COMMENTED** [review]
The text looks good! Would wait for a final review from Edda before merging

File: frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx
Comment (andchiind) [inline frontend/src/components/Pages/FrontPage/MissionOverview/StopDialogs.tsx:115]:
```suggestion
                <Dialog.CustomContent><DialogContent missionTaskType={missionTaskType} /></Dialog.CustomContent>
```

**Review by andchiind — APPROVED** [review]
LGTM

**Pull request merged**
